android-package/build.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:4.1.2"
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven-publish'
+
+group = projectGroup
+version = projectVersion
+
+afterEvaluate {
+    publishing {
+        publications {
+            mavenRelease(MavenPublication) {
+                from components.release
+            }
+        }
+        /* publish to local build/repo folder */
+        repositories {
+            maven {
+                url "$buildDir/repo"
+            }
+        }
+    }
+}
+
+android {
+    compileSdkVersion 30
```

Comment:
if I am not mistaken this one is pretty new, I suppose they are backward compatible, so does it make sense to set it to smth old?

Reply:
23 sound good?